### ISSUE TITLE
Hyperparams and setup for DIORA with random word embeddings on MSCOCO

### ISSUE THREAD

**Issue by roger-tseng** [bobwan1995/cliora#1]
Hi Bo Wan, 

Thanks for making your implementation publicly available! Could you also share the hyperparameters and setup you used for DIORA on MSCOCO without pretrained embeddings? Your work is the first I've seen that shows DIORA working on random word embeddings, and I believe many would benefit from the reproducibility of those results.

Thanks in advance!
Roger

**Comment by bobwan1995** [comment]
Thanks for your interest in this work! The hyperparameters for MSCOCO are shown as below:

> python cliora/scripts/train.py \
    --cuda \
    --max_epoch 30 \
    --arch mlp \
    --batch_size 32 \
    --emb none \
    --hidden_dim 400 \
    --k_neg 100 \
    --log_every_batch 100 \
    --lr 5e-3 \
    --normalize unit \
    --reconstruct_mode softmax \
    --train_filter_length 40 \
    --data_type coco \
    --train_path ./coco_data/train_gold_caps.json \
    --validation_path ./coco_data/val_gold_caps.json \
    --experiment_path  $EXP_PATH

The most important parameters are batch size and lr. The lr is larger than that used in Flickr30K. The json files are directly from VPCFG. I'll supplement the exps of MSCOCO soon.
